### DISCUSSION TITLE
NumberFormatter not found after login

### DISCUSSION THREAD

**Discussion by NielsHamelink-web** [bagisto/bagisto#6099] [Q&A] [answered]
# Bug Report

## Issue Description
After I completed the installation I got the error `Class 'NumberFormatter' not found`.

## Preconditions
- PHP: 8.1

## Bagisto Version
Downloaded from the website, I've run a `composer update` just now.

## Steps To Reproduce
- Download package from the website
- Walk through the installation form
- Login

So note: I have not installed it through `composer create-project bagisto/bagisto`.

## Actual Result
![image](https://user-images.githubusercontent.com/67690385/158289288-eee4fa2f-a110-4ec6-845a-5047829ab84b.png)

**Comment by jitendra-webkul** [comment]
Enable/Install php_intl extension, issue will be resolved.

**Comment by Sanjaybhattwebkul** [comment]
@NielsHamelink-web 
Hope your **issue will be resolved**. so I am **closing** this issue.
If you have still the same issue. you can **reopen** this issue again or can **raise ticket [Here](https://bagisto.uvdesk.com/en/customer/create-ticket/)**

**Comment by NielsHamelink-web** [comment]
> Enable/Install php_intl extension, issue will be resolved.

I have installed and enabled the extension, restarted my PHP server and installed Bagisto again. I still got the same error I got before.

**Comment by Sanjaybhattwebkul** [comment]
@NielsHamelink-web 
Please raise a ticket [Here](https://bagisto.uvdesk.com/en/customer/create-ticket/).
We will provide a better solution over the ticket.

**Comment by Sanjaybhattwebkul** [comment]
@NielsHamelink-web 
Check your **php_intl extension**, in **php info**. 
Please take a reference from [Here](https://prnt.sc/WLtdY9Wayp9y).

**Comment by Sanjaybhattwebkul** [comment]
Hi there,

The **intl extension** should be **enabled** on your **php.ini file.**
If **intl extension** is  not enabled then **just add this**

**_[extension=intl ] in php.ini_**

**Comment by NielsHamelink-web** [comment]
> Hi there,
> 
> The **intl extension** should be **enabled** on your **php.ini file.** If **intl extension** is not enabled then **just add this**
> 
> **_[extension=intl ] in php.ini_**

Like mentioned above I've enabled the extension. I've run `phpenmod intl` and restarted the server. Does not make a difference.
![image](https://user-images.githubusercontent.com/67690385/158399233-925af98c-0533-4e6e-98db-10e952dd42e9.png)

**Comment by NielsHamelink-web** [comment]
To be clear, this is after installation and logging in on the dashboard page of the admin.

**Comment by devansh-webkul** [comment]
Hi @NielsHamelink-web,

These error which you stated is due to intl extension please check intl extension.

**Comment by NielsHamelink-web** [comment]
> Hi @NielsHamelink-web,
> 
> These error which you stated is due to intl extension please check intl extension.

I've commented a screenshot of my phpinfo above... I think you can see that it's enabled like I already told you guys 3 times.

**Comment by devansh-webkul** [comment, marked as answer]
There is nothing we can do here, please check your instance. The problem is pretty common.

Maybe your PHP version and intl version are mismatching. I am just guessing but you need to check this because this is a common known issue.

Reference Link:
https://github.com/bagisto/bagisto/issues/6050
https://github.com/bagisto/bagisto/issues/5212
https://github.com/bagisto/bagisto/issues/4012
